src/renderers/WebGLRenderer.js

```diff
@@ -1359,6 +1363,11 @@ class WebGLRenderer {
 
 			const currentRenderTarget = _this.getRenderTarget();
 			_this.setRenderTarget( _transmissionRenderTarget );
+
+			_this.getClearColor( _currentClearColor );
+			_currentClearAlpha = _this.getClearAlpha();
+			if ( _currentClearAlpha < 1 ) _this.setClearColor( 0x7f7f7f, 0.8 );
```

Comment:
Based on my experiments, values greater than `0x7f7f7f` are too bright. This seems to be a good compromise.

The clear alpha value of 0.8 allows the CSS background to show through when the scene background is transparent. This is a subjective value.